packages/extension-youtube/src/utils.ts

```diff
@@ -50,6 +51,7 @@ export const getEmbedUrlFromYoutubeUrl = (options: GetEmbedUrlOptions) => {
     playlist,
     progressBarColor,
     startAt,
+    sameChannelRel,
```

Comment:
I'd call this `rel: boolean` or `relatedVideos: boolean` and allow to pass `rel={Number(rel)}` so the user can customize it and would be following more the youtube documentation for the embedded player parameters.

Reply:
Sounds good! I've changed it to `rel: number`